unobot.py

```diff
@@ -360,15 +360,22 @@ def render_counts(self, full=NO):
 
     @staticmethod
     def render_cards(bot, cards, who):
+        cards = sorted(cards)
+        wilds = []
+        for card in cards:
+            if 'W' in card:
+                cards.remove(card)
+                wilds.append(card)
+        cards.extend(sorted(wilds))
```

Comment:
With the initial `sorted(cards)` operation, sorting the wilds here again is probably unnecessary except to satisfy my obsession with making absolutely sure that the order is always, _always_, **always** consistent…